using-valtimo/tasks/tasks.md

```diff
@@ -0,0 +1,23 @@
+# Tasks
+
+Tasks are the user task instances that are generated within a process and should be completed by a human actor.
```

Comment:
This line feels very technical. i think we could expand upon it and write it in a way the average person could understand it.